### PR TITLE
test(cli): add global setup and teardown

### PR DESCRIPTION
**Pull request by y-lakhdar**
https://coveord.atlassian.net/browse/CDX-150

### CODE REVIEW

File: packages/cli-e2e/setup.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import type {Browser} from 'puppeteer';
+import {connectToChromeBrowser} from './utils/browser';
+import {clearKeychain, loginWithOffice} from './utils/login';
+
+async function clearChromeBrowsingData(browser: Browser) {
+  const pages = await browser.pages();
+
+  const pageClearDataPromise = [];
+  for (const page of pages) {
```
Comment (louis-bompart) [inline packages/cli-e2e/setup.ts:9]: Do we need to do it on all pages 🤔 ?

**Review by louis-bompart — APPROVED** [review]
Looks good!

File: packages/cli-e2e/setup.ts (hunk 1 above)
Comment (y-lakhdar) [reply packages/cli-e2e/setup.ts:9]: Tbh, I am not sure. I stupidly applied it to all pages just in case. :S

**Review by olamothe — APPROVED** [review]

File: packages/cli-e2e/tsconfig.json [hunk 2]
```diff
@@ -7,6 +7,7 @@
     "skipLibCheck": true,
     "sourceMap": true,
     "forceConsistentCasingInFileNames": true,
-    "types": ["jest"]
+    "types": ["jest"],
+    "lib": ["dom", "es2017"]
```
Comment (y-lakhdar) [inline packages/cli-e2e/tsconfig.json:11]: @louis-bompart, this additional line is required for the `isElementClickable` function

**Pull request merged**